SerialPrograms/Source/CommonFramework/OCR/TesseractPA.h

```diff
@@ -21,22 +21,18 @@
 #define TESSERACT_EXPORT __declspec(dllexport)
 #else
 #define TESSERACT_EXPORT __declspec(dllimport)
-#endif
-
-#else
-
-#define TESSERACT_EXPORT __attribute__((visibility("default")))
```

Comment:
How does this work? If you remove the `TESSERACT_EXPORT` macro, then line 41 won't compile?

Reply:
It's difficult to explain the change with so many ifdef in the code. I have added a new commit to make the change  easier to understand.